interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,67 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+
+//Function do translate from another language to English
```

Comment:
Interesting. So, this is basically trying to get the constant in the reverse direction?